### PR TITLE
Fixes #1988 - fixes build for #1983

### LINKED ISSUE #1988: Fix build
Related to #1976 and #1846.
Needs to be merged before #1983, so the build can pass.

### PR DESCRIPTION
**Pull request by zoepage**
This PR fixes the ES6 template issue with the build dependency.
Please merge before #1983 

r? @miketaylr

### CODE REVIEW

File: webcompat/static/js/lib/bugform.js [hunk 1]
```diff
@@ -221,49 +221,61 @@ function BugForm() {
...
-        } else {
+              })
+              .then(function(issues) {
+                if (issues.items) {
+                  $("#bugListWrapper").show();
+                  $("#bugList").empty();
+                  for (var item in issues.items) {
+                    var issue = issues.items[item];
+                    $("#bugList").append(
+                      "<li><a href='https://webcompat.com/issues/" +
```
Comment (miketaylr) [inline webcompat/static/js/lib/bugform.js]: I already left some feedback about this in the last PR, I think using `_.template` from lodash would be better here. No hard objections in a proof of concept branch, but it should be fixed before it gets merged into master.

**Comment by miketaylr** [comment]
I think @MDTsai should review, since he authored the original stuff.

r? @MDTsai

**Comment by MDTsai** [comment]
Looks good, as merged into master, I think 2 things to do:
- [ ] change search end point from github search API to ours?
- [ ] alternative of fetch for more compatibility.

File: webcompat/static/js/lib/bugform.js (hunk 1 above)
Comment (zoepage) [reply webcompat/static/js/lib/bugform.js]: I agree. Will add this to the follow up issue.

**Comment by zoepage** [comment]
@MDTsai I agree. Will add this to the follow up issue when we merge to master.

**Pull request merged**